src/nvim/ex_cmds_defs.h

```diff
@@ -91,6 +92,7 @@ typedef struct exarg exarg_T;
 #define BAD_DROP        (-2)    // erase it
 
 typedef void (*ex_func_T)(exarg_T *eap);
+typedef int (*ex_cmdpreview_func_T)(exarg_T *eap, long cmdpreview_ns, handle_T cmdpreview_bufnr);
```

Comment:
- The `ex_` part already signals that this is related to commands.
- Seems like the "cmdpreview" name (instead of "preview") is not being used much elsewhere in this PR.

```suggestion
typedef int (*ex_func_preview_T)(exarg_T *eap, long preview_ns, handle_T preview_bufnr);
```

Reply:
Let's make it `ex_preview_func_T` instead to be consistent with `func` and `preview_func` in `ex_cmds.lua`